.ci/jobs.t/elastic+elasticsearch+periodic+java-fips-matrix.yml

```diff
@@ -4,6 +4,6 @@ vars:
   - job-name: elastic+elasticsearch+%BRANCH%+periodic+java-fips-matrix
   - job-display-name: "elastic / elasticsearch # %BRANCH% - java fips compatibility matrix"
   - job-description: "Testing of the Elasticsearch %BRANCH% branch java FIPS compatibility matrix.\n"
-  - matrix-yaml-file: ".ci/matrix-runtime-javas.yml"
+  - matrix-yaml-file: ".ci/matrix-runtime-javas-fips.yml"
```

Comment:
Happy to get any feedback for tackling this in a different way. I'm not aware of how we could further filter the values from the existing matrix-runtime-javas.yml and keeping a specific file for these matrix jobs sounded like a good idea nevertheless. I understand we can define the job entirely here and add the runtime javas in an axis, let me know if this is a preferable approach